libs/cardano-ledger-core/src/Cardano/Ledger/Core.hs

```diff
@@ -235,6 +235,11 @@ class
   getTotalRefundsTxBody pp lookupStakingDeposit lookupDRepDeposit txBody =
     getTotalRefundsTxCerts pp lookupStakingDeposit lookupDRepDeposit (txBody ^. certsTxBodyL)
 
+  -- | This function is not used in the ledger rules. It is only used by the downstream
+  -- tooling to figure out how many witnesses should be supplied for Genesis keys.
+  genesisKeyHashCount :: TxBody era -> Int
```

Comment:
Do we have a convention for when to use functions prefixed by `get` in these typeclasses? I have seen a few cases where the version that didn't start with `get` was deprecate din favour of the `get`-prefixed one  (for example `minFee` and `getMinFree`). 
Is it perhaps to do with the comment that you added that it's not used in the rules?

Reply:
`minfee` used to be a standalone function. I was adding a type family suffix and prefixing with `get` every getter fucntionthat became a type class function, i.e. became era agnostic, like `getMinFeeTx` for example.

I think for consistency sake this one should be also named `getGenesisKeyHashCountTxBody`. 

That being said, there was no official convention enforced for naming. Which I think we should get together at some point and develop as a team.